### PR TITLE
Blog: Modern apps part 1

### PR DESCRIPTION
**Pull request by psiinon**
<details><summary>Screenshot</summary>
<p>

Screenshot 2023-11-02 at 14-57-14 Handling Modern Web Apps Better - Part 1

</p>
</details> 

Ready for review but should not be merged until Nov 3rd.

### CODE REVIEW

File: site/content/blog/2023-11-03-handling-modern-web-apps-better-part1/index.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+The add-on adds 3 new tabs to ZAP which give you a lot more information about what is happening in the browser.
+
+All of this information is available via Browser dev tools, but only for the pages being displayed.
+ZAP shows you all of the essential details for the whole app, and this information is also available when automating ZAP.
+
+#### Client Map
+
+The [Client Map](/docs/desktop/addons/client-side-integration/#client-map) is a representation of the apps that the browser accesses, so superficially it looks similar to the [Sites Tree](/docs/desktop/start/features/sitestree/).
+
+However if you look closely you will see that it actually includes the [URL fragments](https://en.wikipedia.org/wiki/URI_fragment) which are not sent from the browser to the target app, so ZAP doesnt not normally see these.
```
Comment (kingthorin) [inline site/content/blog/2023-11-03-handling-modern-web-apps-better-part1/index.md:44]:
```suggestion
However if you look closely you will see that it actually includes the [URL fragments](https://en.wikipedia.org/wiki/URI_fragment) which are not sent from the browser to the target app, so ZAP doesn't normally see these.
```

File: site/content/blog/2023-11-03-handling-modern-web-apps-better-part1/index.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+The [Client Details](/docs/desktop/addons/client-side-integration/#client-details) tab (also shown above) shows details about the node selected in the Client Map.
+
+It allows you to understand much more about the functionality that this URL represents in the target app.
+
+The types of data displayed include:
+
+* Button: Buttons detected in the DOM
+* Cookies: Cookies set in the browser
+* FORM: Forms detected in the DOM
```
Comment (kingthorin) [inline site/content/blog/2023-11-03-handling-modern-web-apps-better-part1/index.md:67]: I don't have a quick way to check right this minute but is FORM really full caps vs the others?

Purposefully?

Comment (psiinon) [reply site/content/blog/2023-11-03-handling-modern-web-apps-better-part1/index.md:67]: Yes, thats how it appears in the GUI. Can't remember why off hand, but to change that would be a code change..

Comment (kingthorin) [reply site/content/blog/2023-11-03-handling-modern-web-apps-better-part1/index.md:67]: Alright leave it for now :wink:

File: site/content/blog/2023-11-03-handling-modern-web-apps-better-part1/index.md (hunk 1 above)
Comment (psiinon) [reply site/content/blog/2023-11-03-handling-modern-web-apps-better-part1/index.md]: Fixed :)

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**